ros/src/computing/perception/detection/lidar_tracker/packages/lidar_imm_ukf_pda_track/launch/imm_ukf_pda_track.launch

```diff
@@ -19,7 +19,7 @@
 
   <arg name="merge_distance_threshold" default="0.5"/>
 
-  <arg name="namespace" default="/detection/object_tracker/"/>
+  <arg name="namespace" default="/detection"/>
```

Comment:
We have our own launch files (aw_platform) so that don't have to change stuff like this in the core repo. Just change the arg when you include this file from aw_platform.